Source/player.h

```diff
@@ -210,6 +210,15 @@ struct PlayerAnimationData {
 	}
 };
 
+struct SpellCastInfo {
+	spell_id spellId;
+	spell_type spellType;
+	/* @brief Inventory location for scrolls */
+	int8_t spellFrom;
+	/* @brief Used for spell level */
+	int spellLevel;

```

Comment:
spell level is fairly limited in range (0-22 afaik), so probably can use a much smaller size type.

Reply:
That's true and i think we should do this. 🙂 
But I wouldn't include this in this pr.
The pr is big enough and in master `spellLevel` is also an `int`.